### DISCUSSION TITLE
Azure Linux for bare metal host

### DISCUSSION THREAD

**Discussion by Kumareshan009** [microsoft/azurelinux#8607] [General]
I want to deploy Azure Linux on bare metal, the instructions describe the step deploy on Hyper-v enabled VM, Can you please point me to steps to deploy on bare metal ?

**Comment by ghost** [comment]
based on what I'm seeing, Azure Linux has been optimized to the point that it will not easily interface with hardware outside of a Hyper-V container
I believe it can be installed on [Hyper-V Server 2019](https://www.microsoft.com/en-us/evalcenter/evaluate-hyper-v-server-2019), which is available for no cost. I'd like to look more into if this is supported and if doing so has more stringent system requirements then Hyper-V Server has alone.

**Comment by mfrw** [comment]
@Kumareshan009 are you facing any issues while installing from the [ISO](https://aka.ms/mariner-2.0-x86_64-iso) on a baremetal ?
